auto-generated/api-testing/vfadd.c

```diff
@@ -244,3 +244,243 @@ vfloat64m8_t test_vfadd_vf_f64m8_m(vbool8_t mask, vfloat64m8_t op1, float64_t op
+  return __riscv_vfadd_vv_f64m8_rm(op1, op2, __RISCV_FRM_RNE, vl);
+}
+
+vfloat64m8_t test_vfadd_vf_f64m8_rm(vfloat64m8_t op1, float64_t op2, size_t vl) {
+  return __riscv_vfadd_vf_f64m8_rm(op1, op2, __RISCV_FRM_RNE, vl);
+}
+
+vfloat16mf4_t test_vfadd_vv_f16mf4_rm_m(vbool64_t mask, vfloat16mf4_t op1, vfloat16mf4_t op2, size_t vl) {
+  return __riscv_vfadd_vv_f16mf4_rm_m(mask, op1, op2, __RISCV_FRM_RNE, vl);
+}
+
+vfloat16mf4_t test_vfadd_vf_f16mf4_rm_m(vbool64_t mask, vfloat16mf4_t op1, float16_t op2, size_t vl) {
+  return __riscv_vfadd_vf_f16mf4_rm_m(mask, op1, op2, __RISCV_FRM_RNE, vl);
+}
+
+vfloat16mf2_t test_vfadd_vv_f16mf2_rm_m(vbool32_t mask, vfloat16mf2_t op1, vfloat16mf2_t op2, size_t vl) {
+  return __riscv_vfadd_vv_f16mf2_rm_m(mask, op1, op2, __RISCV_FRM_RNE, vl);
+}
+
+vfloat16mf2_t test_vfadd_vf_f16mf2_rm_m(vbool32_t mask, vfloat16mf2_t op1, float16_t op2, size_t vl) {
+  return __riscv_vfadd_vf_f16mf2_rm_m(mask, op1, op2, __RISCV_FRM_RNE, vl);
+}
+
+vfloat16m1_t test_vfadd_vv_f16m1_rm_m(vbool16_t mask, vfloat16m1_t op1, vfloat16m1_t op2, size_t vl) {
+  return __riscv_vfadd_vv_f16m1_rm_m(mask, op1, op2, __RISCV_FRM_RNE, vl);
+}
+
+vfloat16m1_t test_vfadd_vf_f16m1_rm_m(vbool16_t mask, vfloat16m1_t op1, float16_t op2, size_t vl) {
+  return __riscv_vfadd_vf_f16m1_rm_m(mask, op1, op2, __RISCV_FRM_RNE, vl);
+}
+
+vfloat16m2_t test_vfadd_vv_f16m2_rm_m(vbool8_t mask, vfloat16m2_t op1, vfloat16m2_t op2, size_t vl) {
+  return __riscv_vfadd_vv_f16m2_rm_m(mask, op1, op2, __RISCV_FRM_RNE, vl);
+}
+
+vfloat16m2_t test_vfadd_vf_f16m2_rm_m(vbool8_t mask, vfloat16m2_t op1, float16_t op2, size_t vl) {
+  return __riscv_vfadd_vf_f16m2_rm_m(mask, op1, op2, __RISCV_FRM_RNE, vl);
+}
+
+vfloat16m4_t test_vfadd_vv_f16m4_rm_m(vbool4_t mask, vfloat16m4_t op1, vfloat16m4_t op2, size_t vl) {
+  return __riscv_vfadd_vv_f16m4_rm_m(mask, op1, op2, __RISCV_FRM_RNE, vl);
+}
+
+vfloat16m4_t test_vfadd_vf_f16m4_rm_m(vbool4_t mask, vfloat16m4_t op1, float16_t op2, size_t vl) {
+  return __riscv_vfadd_vf_f16m4_rm_m(mask, op1, op2, __RISCV_FRM_RNE, vl);
+}
+
+vfloat16m8_t test_vfadd_vv_f16m8_rm_m(vbool2_t mask, vfloat16m8_t op1, vfloat16m8_t op2, size_t vl) {
+  return __riscv_vfadd_vv_f16m8_rm_m(mask, op1, op2, __RISCV_FRM_RNE, vl);
+}
+
+vfloat16m8_t test_vfadd_vf_f16m8_rm_m(vbool2_t mask, vfloat16m8_t op1, float16_t op2, size_t vl) {
+  return __riscv_vfadd_vf_f16m8_rm_m(mask, op1, op2, __RISCV_FRM_RNE, vl);
+}
+
+vfloat32mf2_t test_vfadd_vv_f32mf2_rm_m(vbool64_t mask, vfloat32mf2_t op1, vfloat32mf2_t op2, size_t vl) {
+  return __riscv_vfadd_vv_f32mf2_rm_m(mask, op1, op2, __RISCV_FRM_RNE, vl);
+}
+
+vfloat32mf2_t test_vfadd_vf_f32mf2_rm_m(vbool64_t mask, vfloat32mf2_t op1, float32_t op2, size_t vl) {
+  return __riscv_vfadd_vf_f32mf2_rm_m(mask, op1, op2, __RISCV_FRM_RNE, vl);
+}
+
+vfloat32m1_t test_vfadd_vv_f32m1_rm_m(vbool32_t mask, vfloat32m1_t op1, vfloat32m1_t op2, size_t vl) {
+  return __riscv_vfadd_vv_f32m1_rm_m(mask, op1, op2, __RISCV_FRM_RNE, vl);
```

Comment:
should be `__riscv_vfadd_vv_f32m1_m_rm` instead of `__riscv_vfadd_vv_f32m1_rm_m`?

Reply:
We are having `_rm_m` for the symmetry of `__riscv_vfadd_vv_f32m1` and `__riscv_vfadd_vv_f32m1_rm` as two sets (the implicit-`frm` intrinsics and the explicit-`frm` intrinsics), and then the policy variants be applied upon these two sets.

Reply:
Just updated the description, thank you for noticing.

Reply:
Cool, thank you for confirmation.